go.mod

```diff
@@ -5,10 +5,10 @@ go 1.17
 require (
 	github.com/Jeffail/gabs/v2 v2.6.1
 	github.com/ThreeDotsLabs/watermill v1.1.1
-	github.com/eclipse-kanto/suite-connector v0.1.0-M1.0.20220822083435-d47b0a2edf1a
+	github.com/eclipse-kanto/suite-connector v0.1.0-M2.0.20230222081206-577d4deaa329
 	github.com/imdario/mergo v0.3.12
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.8.1
 	go.etcd.io/bbolt v1.3.6
```

Comment:
I do not see _go.etcd.io/bbolt_ in the notice file. Could you please check whether it should be added.

Reply:
nice catch, it should be added, `go version -m local-digital-twins` returns:
```                                                                                                                                                                                                                             
local-digital-twins/local-digital-twins: go1.19.5
        path    github.com/eclipse-kanto/local-digital-twins/cmd/twins
        mod     github.com/eclipse-kanto/local-digital-twins    (devel)
        dep     github.com/Jeffail/gabs/v2      v2.6.1  h1:wwbE6nTQTwIMsMxzi6XFQQYRZ6wDc1mSdxoAN+9U4Gk=
...
        dep     go.etcd.io/bbolt        v1.3.6  h1:/ecaJf0sk1l4l6V4awd65v2C3ILy7MSj+s/x1ADCIMU=
...
        build   vcs.time=2023-02-22T11:25:07Z
        build   vcs.modified=false
```
Its out of scope of this PR, but I guess we can use it